Examples/Run/Reconstruction/Common/RecCKFTracks.cpp

```diff
@@ -146,7 +189,7 @@ int runRecCKFTracks(int argc, char* argv[],
   trackFindingCfg.inputMeasurements = hitSmearingCfg.outputMeasurements;
   trackFindingCfg.inputSourceLinks = hitSmearingCfg.outputSourceLinks;
   trackFindingCfg.inputInitialTrackParameters =
-      particleSmearingCfg.outputTrackParameters;
+      trackParams(estimateTrackParamsFromSeed);
```

Comment:
As mentioned above, here I do not know anymore what this does exactly.

Reply:
Fixed.